pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -62,7 +62,7 @@ func (q *graceTerminateRSList) add(rs *listItem) bool {
 		return false
 	}
 
-	klog.V(5).Infof("Adding rs %v to graceful delete rsList", rs)
+	klog.V(5).InfoS("Adding rs graceful delete rsList", "realServer", rs)
```

Comment:
"Adding rs item to graceful delete rsList" sounds better and will be make it easier to understand the intent for the debugger.

Reply:
Thanks for pointing out, fixed it.